### ISSUE TITLE
Delete/Deprecate conan-plibsys - upstream maintainer adopted

### ISSUE THREAD

**Issue by solvingj** [bincrafters/community#593]
This ticket is just to mark the deletion of plibsys which has been adopted by the library maintainer.  

https://github.com/saprykin/conan-plibsys

**Comment by Croydon** [comment]
Why got the repo deleted? We agreed on a deprecating process before which doesn't include deleting?

**Comment by solvingj** [comment]
Yes, indeed, that is the policy we should be following in most cases.  I considered this an exception, but in the future I'll open the ticket and allow time for discussion first.   The reason I considered this an exception was that I intended to delete it over 6 months ago, right after the upstream maintainer got his package working and got it accepted into Conan Center.  Our package had only existed for a few weeks at that point, was not in Conan Center, and I was confident nobody was using it.  When I noticed our "old" version and repository was still lingering, I just saw it as fixing a mistake and didn't think twice about it. Will be more deliberate in the future, even with non-conan-center packages.  Thanks for mentioning it.
